### PR TITLE
Optional separate image paths for each platform

### PR DESCRIPTION
**Pull request by mdudek**
Implemented support for custom config section name.  It's possible to have separate config sections for iOS/Android platform.

Example:

pubspec.yaml:
```yaml
flutter_icons_android:
  image_path: "resources/appIcon_android.png"
  android: true

flutter_icons_ios:
  image_path: "resources/appIcon_iOS.png"
  ios: true

```
commands: 
```
flutter pub pub run flutter_launcher_icons:main flutter_icons_android
flutter pub pub run flutter_launcher_icons:main flutter_icons_ios
```

### CODE REVIEW

**Comment by MarkOSullivan94** [comment]
Hi @mdudek, thanks for your contribution!

I want to avoid our users from having to run multiple commands but I love having different paths for images which was suggested (see #18). 

If you're interested in adapting your PR, what I'd like to see instead:

- image_path -> This is where you'd specify an image path for both iOS and Android icons
- image_path_android -> This is where you'd specify an image path for just Android icons
- image_path_ios -> This is where you'd specify an image path for just iOS icons

Let me know if you're happy to work on this.

**Comment by mdudek** [comment]
Hi @MarkOSullivan94 ,
I have made changes to support platform specific icon paths in config as you suggested:
```yaml
flutter_icons:
  image_path_android: "resources/appIcon_android.png"
  image_path_ios: "resources/appIcon_iOS.png"
  android: true
  ios: true
```

Specific path is used instead of general 'image_path' if defined.

Tests are also updated.

File: README.md [hunk 1]
```diff
@@ -35,13 +35,17 @@ dev_dependencies:
...
+  image_path: "icon/icon.png"
+  image_path_android: "icon/icon_android.png"
+  image_path_ios: "icon/icon_ios.png"
   android: true
   ios: "Example-Icon"
 ```
 ```
 #### Attributes: 
 image_path: The location of the icon image file which you want to use as the app launcher icon
+image_path_android: The location of the icon image file specific for Android platform (optional)
```
Comment (MarkOSullivan94) [inline README.md]: Perhaps include some further information like between line 50-54 that the users don't need all 3.

File: example/pubspec.yaml [hunk 2]
```diff
@@ -18,6 +18,8 @@ dev_dependencies:
 
 flutter_icons:
   image_path: "assets/images/icon-710x599.png"
+  image_path_android: "assets/images/icon-710x599-android.png"
+  image_path_ios: "assets/images/icon-710x599-ios.png"
```
Comment (MarkOSullivan94) [inline example/pubspec.yaml]: Do both icon-710x599-android.png and icon-710x599-ios.png exist? If not, ensure they both exist. Example should allow anyone to test out the package quickly (myself included).

File: lib/main.dart [hunk 3]
```diff
@@ -29,8 +31,8 @@ Future<Map> loadConfigFile(String path) async {
   return config;
 }
 
-Map loadFlutterIconsConfig(Map config) {
-  return config["flutter_icons"];
+Map loadFlutterIconsConfig(Map config, String configSectionName) {
```
Comment (MarkOSullivan94) [inline lib/main.dart]: Is this needed?

**Comment by MarkOSullivan94** [comment]
@mdudek thanks for making the changes! Looking forward to getting these changes merged in.

I'm very busy this weekend so it might not be until next week before I get a proper chance to sit down and review it more.

File: lib/main.dart [hunk 4]
```diff
@@ -5,9 +5,11 @@ import 'package:dart_config/default_server.dart';
 
 createIcons(List<String> arguments) async {
   loadConfigFile("pubspec.yaml").then((Map yamlConfig) {
-    Map config = loadFlutterIconsConfig(yamlConfig);
+    var configSectionName = arguments.length == 1 ? arguments[0] : "flutter_icons";
```
Comment (MarkOSullivan94) [inline lib/main.dart]: Revert this change back to `Map config = loadFlutterIconsConfig(yamlConfig);`

**Comment by mdudek** [comment]
Done ;-)

**Comment by MarkOSullivan94** [comment]
Tested your local branch's example project to ensure everything was working as expected. Also had a look through the code and everything looks good to me!
